components/plugins/lib.rs

```diff
@@ -70,5 +70,4 @@ fn register_clippy(reg: &mut Registry) {
 }
 #[cfg(not(feature = "clippy"))]
 fn register_clippy(reg: &mut Registry) {
```

Comment:
No, it's still necessary. We don't `#![plugin(clippy)]` on all our crates, we just register it here as if the clippy lints are our own.